packages/swarm-styles/lib/components/select.css

```diff
@@ -0,0 +1,61 @@
+[data-swarm-select-wrapper] {
+    position:relative;
+}
+
+[data-swarm-select] label span {
+    color: var(--color-gray-5);
+}
+
+[data-swarm-select-helper-text] {
+    font-size: var(--font-size-small);
+}
+
+[data-swarm-select] select {
+    appearance: none;
+    user-select: none;
+    padding-top: var(--size-space-default);
+    padding-right: var(--size-space-double);
+    padding-bottom: var(--size-space-default);
+    padding-left: var(--size-space-default);
+    width: 100%;
+    border: 1px solid var(--color-gray-3);
+    /* ToDo: do we need a box shadow? do we want it on focus or on error states as well? */
```

Comment:
This is where I wasn't sure if we needed a box shadow so i just commented it out